geneva.spec

```diff
@@ -0,0 +1,32 @@
+### RPM external geneva 1.0-RC3
+Source: git+https://stash.desy.de/scm/geneva/geneva-public.git?obj=master/%{realversion}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
+
+BuildRequires: cmake
+
+Requires: python py2-setuptools py2-numpy gsl boost lhapdf hepmc openloops pythia8
+
+%prep
+%setup -q -n %{n}-%{realversion}
+
+mkdir build
```

Comment:
@mkirsano , can you please 
 - move the `%build` statement just above this line
 - create build directory outside of the source tree e.g something like https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_10_3_X/gcc700/root.spec#L35 . You also need to update cmake to use ../%{n}-%{realversion}